docs/source/guide/ddd-1-intro.myst

```diff
@@ -12,3 +12,112 @@ kernelspec:
 ---
 
 # How do I use DDD?
+
+Digital Dynamical Decoupling (DDD) is an error mitigation technique in which
+sequences of gates are applied to slack windows, i.e. single-qubit idle
+windows, in a quantum circuit. Mitiq provides standard DDD sequences that can
+be directly imported from `mitiq.ddd.rules`. 
```

Comment:
```suggestion
windows, in a quantum circuit. Such sequences of gates can reduce the coupling
between the qubits and the environment, mitigating the effects of noise.
```
I would avoid mentioning ddd.rules in the first sentence. Better mentioning the scope of ddd sequences.